packages/ui/src/hooks/useTimeSince.ts

```diff
@@ -0,0 +1,19 @@
+import { useEffect, useState } from 'react'
+import dayjs from 'dayjs'
+import relativeTime from 'dayjs/plugin/relativeTime'
+
+dayjs.extend(relativeTime)
+
+export default function (timestamp?: number) {
```

Comment:
wonder if this really needs to be a hook. maybe nice to not do the conversion every time. could use a memo hook as well

```js 
const useTimeSince = (timestamp) => ( 
  useMemo(() => dayjs.unix(timestamp).fromNow(), [timestamp])
)